lib/modules/manager/mix/artifacts.ts

```diff
@@ -26,15 +29,51 @@ export async function updateArtifacts({
   config,
 }: UpdateArtifact): Promise<UpdateArtifactsResult[] | null> {
   logger.debug(`mix.getArtifacts(${packageFileName})`);
-  if (updatedDeps.length < 1) {
-    logger.debug('No updated mix deps - returning null');
+  const isLockFileMaintenance = config.updateType === 'lockFileMaintenance';
```

Comment:
@viceice is this your recommended way?

Reply:
- https://github.com/renovatebot/renovate/pull/33971